.github/workflows/ci.yml

```diff
@@ -65,18 +65,21 @@ jobs:
         with:
           fetch-depth: 0
 
-      - uses: ammaraskar/sphinx-action@master
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
         with:
-          docs-folder: "docs/"
-          pre-build-command: |
-            apt update --yes && apt install --yes git build-essential pandoc
-            pip install -U pip setuptools wheel
-            pip install -e .[docs]
-            git describe --tags
-            python -c 'import ctapipe; print(ctapipe.__version__)'
+          python-version: 3.8
```

Comment:
why is it necessary to specify the python version here?

Reply:
Why should we rely on whatever is the current default?

Reply:
I'm asking because just above you use `${{ matrix.python-version }}` which can be also 3.7

Reply:
This is for the unit tests, which we run for multiple python versions. It does not really make sense to build the documentation using multiple python versions.

Reply:
exactly, so it should be "3.8" instead of "${{ matrix.python-version }}" (not a problem, though)

Reply:
I don't understand.

Reply:
yes, but line 68 refers to the `docs` job, not to the `tests` one, so the name of the job should reflect the version we are using, which is only 3.8, that was my confusion (line 30 instead refers to the tests and is correct)

Reply:
Oh, there was a copy paste leftover above in the `name`. This is only for displaying, it does not influence the build.

Reply:
yes I know, that's why I said it's not a problem, it just triggered a confusion from my side